module1/src/main/java/net/broscorp/checkstyle/StyledClass.java

```diff
@@ -1,7 +0,0 @@
-package net.broscorp.checkstyle;
-
-public class StyledClass {
-    public static void MAIN(String[] args) {
-        System.out.println ("Hello checkstyle");
-    }
-}
```

Comment:
could we keep this file unchanged?

Reply:
@Pakirava-Datsuma , It was allowed to delete this file

Reply:
Ok, I didn't know that